Dockerfile.dev

```diff
@@ -3,10 +3,10 @@ FROM ruby:2.6.6
 RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
     echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
     curl -sL https://deb.nodesource.com/setup_14.x | bash - && \
-    apt-get install -y cmake nodejs yarn
+    apt-get install -y cmake nodejs yarn tmux
 
-RUN curl -L -o - https://github.com/DarthSim/hivemind/releases/download/v1.0.6/hivemind-v1.0.6-linux-arm64.gz | gunzip > /usr/local/bin/hivemind && \	
-    chmod u+x /usr/local/bin/hivemind
+RUN curl -L -o - https://github.com/DarthSim/overmind/releases/download/v2.2.0/overmind-v2.2.0-linux-arm64.gz | gunzip > /usr/local/bin/overmind && \	
+    chmod u+x /usr/local/bin/overmind
```

Comment:
Overmind is hivemind's big sibling.

This allows us to do `docker-compose exec website overmind restart xxx` (where xxx is one of the services that is run via overmind), rather than having to reboot the whole website container.

Reply:
Nice!